src/components/BigNumberPad.js

```diff
@@ -17,27 +18,64 @@ const padNumbers = [
     ['.', '0', '<'],
 ];
 
-const BigNumberPad = props => (
-    <View style={[styles.flexColumn, styles.w100]}>
-        {_.map(padNumbers, (row, rowIndex) => (
-            <View key={`NumberPadRow-${rowIndex}`} style={[styles.flexRow, styles.mt3]}>
-                {_.map(row, (column, columnIndex) => {
-                    // Adding margin between buttons except first column to
-                    // avoid unccessary space before the first column.
-                    const marginLeft = columnIndex > 0 ? styles.ml3 : {};
-                    return (
-                        <ExpensifyButton
-                            key={column}
-                            style={[styles.flex1, marginLeft]}
-                            text={column}
-                            onPress={() => props.numberPressed(column)}
-                        />
-                    );
-                })}
+class BigNumberPad extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {timer: null};
+    }
+
+    /**
+     * Handle long press key on number pad.
+     * Only handles the '<' key and starts the continuous input timer.
+     *
+     * @param {String} key
+     */
+    handleLongPress(key) {
+        // Only handles deleting.
+        if (key !== '<') {
+            return;
+        }
+        this.setState(
+            {
+                timer: setInterval(() => {
+                    this.props.numberPressed(key);
+                }, 100),
```

Comment:
Not very intuitive.
Better set the interval above and then set the handle to State in next line.